packages/qgrid-ngx/src/lib/foot/tf-core.directive.ts

```diff
@@ -21,15 +21,15 @@ export class TfCoreDirective implements DomTd, OnInit, OnDestroy {
   @Input('q-grid-core-tf') columnView: ColumnView;
 
   $implicit = this;
-  element: HTMLElement = null;
+  element: HTMLElement;
```

Comment:
to remove default value, please check if there is no null checks for the element anywhere or hasOwnProperty